coremltools/converters/mil/frontend/torch/torch_op_registry.py

```diff
@@ -127,3 +129,14 @@ def func_wrapper(func):
         # decorator called without argument
         return func_wrapper
     return func_wrapper(_func)
+
+
+def is_torch_fx_node_supported(torch_fx_node: torch.fx.Node) -> bool:
+    if torch_fx_node.op != "call_function":
+        raise ValueError("Only call_function node needs query for supportability in CoreML")
```

Comment:
The error message is not super clear.
What are the cases when the `.op` will not be a `call_function`?  

Should we error out or return false in this case, so as to not error out the whole export process?

Reply:
Yeah return false makes more sense, revised.

There are 5 types of fx node:
1. placeholder
2. get_attr
3. call_function
4. call_module
5. output